webcompat/templates/issue/issue-aside.html

```diff
@@ -1,13 +1,11 @@
-<aside class="wc-Issue-aside">
-  <div class="wc-Issue-information wc-Issue-information--fullWidth" id="js-Issue-aside">
-    {% include "issue/aside.jst" %}
-  </div>
-  <div class="wc-Issue-labels js-Issue-labels">
-    {% include "issue/issue-labels.jst" %}
-    {% include "web_modules/label-editor.jst" %}
-  </div>
-  <div class="wc-Issue-milestones js-Issue-milestones">
-    {% include "issue/issue-milestones.jst" %}
-    {% include "web_modules/milestone-editor.jst" %}
-  </div>
-</aside>
+<div id="js-Issue-aside">
+	{% include "issue/aside.jst" %}
+</div>
+<div class="js-Issue-labels label-box-editor label-box-section">
+	{% include "issue/issue-labels.jst" %}
+	{% include "web_modules/label-editor.jst" %}
+</div>
+<div class="js-Issue-milestones label-box-editor label-box-section">
+	{% include "issue/issue-milestones.jst" %}
+	{% include "web_modules/milestone-editor.jst" %}
+</div>
```

Comment:
Also, please add newlines back to all files you removed them from. Editorconfig will do this for you.